functions.php

```diff
@@ -46,10 +48,14 @@ function get_below_the_fold($post_id=null) {
 			<div class="span4" id="below-the-fold-icon">
 				<p><a href="<?=do_shortcode($link);?>"><img src="<?=THEME_IMG_URL?>/below-the-fold.png" alt="Speak up when you know of or suspect unethical behavior." title="<?=$title?>" /></a></p>
 			</div>
-			<div class="span8" id="below-the-fold-content">
+			<div class="span4" id="below-the-fold-content">
 				<h3><?=$title?></h3>
 				<?=$blurb?>
 			</div>
+			<div class="span4" id="code-of-conduct">
+				<p class="span6"><a href="<?=do_shortcode($ucfcclink);?>" target="_blank"><img src="<?=THEME_IMG_URL?>/UCF-CodeOfConduct.jpg" alt="<?=$ucfcccta?>" title="<?=$ucfcccta?>" /></a></p>
```

Comment:
Since this theme is using Bootstrap 2, these inner `.span6`'s probably need to be `.span2`'s to fit within the parent `.span4` (unless they are wrapped in a `.row-fluid` instead of a `.row`, in which case `.span6`'s should work as expected).  See http://bootstrapdocs.com/v2.2.2/docs/scaffolding.html#gridSystem